bin/addRedirectURItoConnect.ts

```diff
@@ -0,0 +1,54 @@
+import { updateApplication } from "../lib/commands/updateApplication";
+import { getApplication } from "../lib/queries/getApplication";
+import { config } from "../src/config";
+
+const addingRedirectURIToConnect = async (): Promise<any> => {
+  try {
+    if (process.env.GITHUB_CONTEXT_EVENT === undefined)
+      throw new Error("githubContextEvent is undefined");
+
+    const githubActionsContext = JSON.parse(process.env.GITHUB_CONTEXT_EVENT);
+
+    if (githubActionsContext.deployment_status === undefined)
+      throw new Error("deploymentStatus is undefined");
+
+    const vercelDeployment = githubActionsContext.deployment_status;
+
+    if (vercelDeployment.state === "success") {
+      const testApp = await getApplication(config.connectAccountTestAppId).then(
+        (results) => {
+          if (results.errors) {
+            throw results.errors;
+          }
+
+          if (!results.data) {
+            throw new Error("app not found");
+          }
+
+          return results.data.provider.application;
+        },
+      );
+
+      const updatedTestApp = {
+        ...testApp,
+        redirectUris: [...testApp.redirectUris, vercelDeployment.target_url],
+      };
+
+      await updateApplication(updatedTestApp).then((results) => {
+        if (results.errors) {
+          throw results.errors;
+        }
+
+        if (!results.data) {
+          throw new Error("update error");
+        }
+
+        return results.data;
```

Comment:
Are you sure you need to return this :)?

Reply:
Nope, indeed.